src/main/java/org/apache/sysml/api/python/SystemML.py

```diff
@@ -37,11 +37,329 @@
 from pyspark.ml import Estimator, Model
 from scipy.sparse import spmatrix
 from scipy.sparse import coo_matrix
+from contextlib import contextmanager
+
+def simpleAddPyDMLStmt(fnName, args=None, obj=None):
+    m = Matrix(0)
+    addPyDMLStmt(fnName, lhs=m, args=args, obj=obj)
+    return m
+
+def addPyDMLStmt(fnName, lhs=None, args=None, obj=None):
+    pydmlStmt = [getTabs()]
+    if lhs is not None:
+        pydmlStmt = pydmlStmt + [str(lhs.getID()), ' = ']
+    if obj is None:
+        pydmlStmt = pydmlStmt + [ fnName, '(']
+    else:
+        pydmlStmt = pydmlStmt + [  obj, '.', fnName, '(']
+    if args is not None:
+        pydmlStmt = pydmlStmt + args
+    MLContext.globalMLContext.pydml.append(''.join(pydmlStmt + [')\n']))
+ 
+def addPyDMLString(pydmlStmt):
+    MLContext.globalMLContext.pydml.append(''.join(pydmlStmt))
+   
+# Constructor         
+def full(init, rows=None, cols=None):
```

Comment:
Can you update this to match `numpy.full` \[1] to the extent possible?  We should accept a `shape` that can be either a single `int` referring the number of rows (with cols implicitly set to 1) or a `tuple` referring to the rows and cols, as well as an `fill_value`.

\[1]: http://docs.scipy.org/doc/numpy/reference/generated/numpy.full.html